### PR TITLE
upgrade to eslint 6

### PR DESCRIPTION
**Pull request by jimexist**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   ],
   "license": "MIT",
   "private": false,
-  "version": "0.1.1",
+  "version": "0.2.0-0",
```
Comment (ljharb) [inline package.json:19]: typically versions are only bumped in commits direct to master, not in PRs

File: package.json [hunk 2]
```diff
@@ -53,13 +53,13 @@
     "typescript": "^3.5.3"
   },
   "peerDependencies": {
-    "vue-eslint-parser": "~6.0.0",
-    "eslint-plugin-vue": "~5.2.0",
-    "eslint": "~5.16.0"
+    "vue-eslint-parser": "~6.0.4",
+    "eslint-plugin-vue": "~5.2.3",
+    "eslint": "~6.1.0"
```
Comment (ljharb) [inline package.json]: you may want to use ^ instead of ~ throughout?

Comment (ljharb) [reply package.json]: you’ll want eslint to actually be both a peer dep and a runtime dep.

File: package.json (hunk 1 above)
Comment (jimexist) [reply package.json:19]: is this also the case for `prerelease`?

Comment (ljharb) [reply package.json:19]: yes; typically you’d test things by installing fTom git, not prereleases.

All publishes need to be a git tag, prerelease or not.

**Pull request merged**